app/components/icon_list_item_component.html.erb

```diff
@@ -1,6 +1,6 @@
-<li class="usa-icon-list__item">
+<%= content_tag(:li, **tag_options, class: [*tag_options[:class], 'usa-icon-list__item']) do %>
```

Comment:
do we need to remove class so we don't accidentally override it?

```suggestion
<%= content_tag(:li, **tag_options.except(:class), class: [*tag_options[:class], 'usa-icon-list__item']) do %>
```

Reply:
> do we need to remove class so we don't accidentally override it?

The order of the hash is intentional in ensuring that we always override the `class` since it's defined last, and [the relevant spec is written to cover for regressions on this behavior](https://github.com/18F/identity-idp/pull/9555/files#diff-7562222c21ed3c38d3d545509fff1ff6ed9f73c2ef0ddbcb627bb28848eb9d88R79).